docs/get-started/sending-transactions.md

```diff
@@ -35,21 +41,25 @@ Then, increase the base fee in the **Max base fee (gwei)** field, not the Priori
 The resulting transaction fee is 0.0018713 XTZ.
 Etherlink does not use a priority fee (also called a tip) to speed up transactions, so you can ignore the Priority fee field.
 
-:::note
+<img src="/img/metamask-change-fee-custom-base-fee.png" alt="Setting the maximum base fee" style={{width: 300}} />
 
-Any difference between the base fee that you set and the necessary fee for the transaction is refunded to you.
-Etherlink takes only the fee that is necessary to include the transaction at the current base fee, not the maximum that you set for the transaction.
+Now you can send the transaction as usual and accept a higher fee if it is necessary based on the network's base fee.
 
-:::
+### Increasing fees for a new transaction (Rabby wallet)
 
-<img src="/img/metamask-change-fee-custom-base-fee.png" alt="Setting the maximum base fee" style={{width: 300}} />
+To change the fees in the Rabby wallet browser extension, click **Advanced Settings** in the transaction confirmation window:
```

Comment:
This is the wrong way to change the gas price. Should be change from "Fast" to "Custom" like you're doing for MM
<img width="406" alt="image" src="https://github.com/user-attachments/assets/496f2171-66cf-4b64-a1d6-5d0fbfb9a050" />

then amend base fee per gas to, e.g. 10
<img width="403" alt="image" src="https://github.com/user-attachments/assets/d3d2817c-5cc2-4809-9e8c-3b6f6591a758" />
The prio fee will also be set to 10 initially, that will be ignored by the chain

The proposed screens show increasing the gas limit (ie the max permitted gas consumption that the tx can use at the current price)

Reply:
Ah, thanks, first time using Rabby. I have made corrections. One Chrome/Macos I don't see the priority fee in that window. Maybe it's different on different platforms.

Reply:
I'm also using chrome macos on laptop... to reproduce my screen, try sending 0.1 xtz to your own address via Rabby

The one you saw without priority could be down to the dapp - if it's set up to be "legacy" / type 0 tx, you'll only get a "gas price" field, no prio fee.

Reply:
Strange, I'm just sending XTZ directly to another address with the simple "Send" button in Rabby and I still don't see a priority fee.

Reply:
Added a note to ignore the priority fee if it is shown.